pkg/oonf_api/0001-add-RIOT-support.patch

```diff
@@ -133,9 +133,7 @@ index dedab2c..fdc3e82 100644
 +#ifndef RIOT_VERSION
  #include <net/if.h>
 +#else
-+#include "net_help.h"
-+#include "inet_ntop.h"
-+#include "inet_pton.h"
```

Comment:
I didn't tested with `oonf` yet, but doesn't removing the both includes will harm using this package?